src/Model/Button/AccountLink.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace BokaiTid\FBMessenger\Model\Button;
+
+use Tgallice\FBMessenger\Model\Button;
+
+class AccountLink extends Button
+{
+    /**
+     * @var string $url
+     */
+    private $url;
+
+    /**
+     * @param string $url
+     */
+    public function __construct($url)
+    {
+        parent::__construct(Button::TYPE_ACCOUNT_LINK);
+
+        $this->url = $url;
+    }
+
+    /**
+     * @return string
+     */
+    public function getUrl()
+    {
+        return $this->url;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function jsonSerialize()
+    {
+        $json = parent::jsonSerialize();
+        $json['url'] = $this->url;
+
+        return $json;
+    }
+}
```

Comment:
I you want add some modification without create a new commit you can do like that:
```
$ git add <files>
// Instead of create a new commit you can amend the previous one
// the --no-edit flag will keep the previous commit message
$ git commit --amend --no-edit
```

But this is to avoid because it's difficulte to keep track on what is update for reviewers. But no worries here ;)